packages/react-native/src/transport.ts

```diff
@@ -0,0 +1,51 @@
+import { Types, Util } from '@honeybadger-io/core'
+import { Platform  } from 'react-native';
+import * as pkg from '../package.json'
+
+export class Transport implements Types.Transport {
+  
+  async send(
+      options: Types.TransportOptions, 
+      payload: Types.NoticeTransportPayload
+    ): Promise<{ statusCode: number; body: string; }> {
+    
+    payload.notifier = {
+      name: pkg.name, 
+      url: pkg.repository.url, 
+      version: pkg.version,
+    }
+
+    const params = {
+      method: options.method,
+      headers: {
+        ...options.headers, 
+        'User-Agent': this.buildUserAgent()
+      },
+      // TODO: is this sanitize helpful?
```

Comment:
I can't tell for sure here, but generally yes. It's the function we have at the client to sanitize the payload (for example, recursive structures).